demo/mobile/src/app/app.js

```diff
@@ -22,6 +25,112 @@ angular.module('wfm-mobile', [
   }),
   require('@raincatcher/vehicle-inspection')
 ]);
+
+var auth = {};
```

Comment:
Code seems to be perfect but it we want to archieve our target (provide reference implementation for keycloak in demo application) we may need to separate it to different file or even module. 
Code can be loaded before angular even if it's in separate module. Having this code in app.js is antipattern as app.js responsibility is to mount all angular specific modules. There should not be any business logic here.